_posts/events/2021-01-28---how-vulnerable-is-your-azure-environment.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: single
+title: "2021-01-28 - How vulnerable is your Azure environment?"
+date: 2021-01-28 17:00:00 +0000
+comments: true
+published: true
+categories: ["events"]
+tags: ["Events"]
+author: Mike Martin
+---
+
+Trespasser detected! Let's investigate! How did they get in and what really happened: let's see how Azure counter attacks!
+
+## Agenda
+
+### How vulnerable is your Azure environment?
+
+Through discussion and demonstration you’ll gain insight into the following topics: Azure Attack Chain - Methods that hackers use to exploit Azure environments; Identifying critical flaws in your attack surface; Lessons to learn from recent cloud breaches; Preventing attacks to your Azure environments
+
+<img src="/assets/media/speakers/david-okeyde.jpg" alt="David Okeyde" align="left" height="100" width="100" style="margin-right: 20px;">**Speaker: David Okeyde** *David is a cloud security architect at the prisma cloud speedboat at Palo Alto Networks. Before that, he was an independent consultant helping companies secure their Azure environments through private expert level trainings and assessments. He holds 13 professional certifications across Azure and AWS platforms, including the Azure Security Engineer, Azure DevOps and AWS Security Specialist certifications. He has also authored two cloud computing courses for the popular cybersecurity training platform - Cybrary.
+
+David has over a decade of experience in Cybersecurity (consultancy, design, implementation) and over 6 years of experience as a trainer. He has worked with organizations of different sizes from startups to major enterprises to government organizations.
+
+David has developed multiple vulnerable by design automation templates that can be used to practice cloud penetration testing techniques. He regularly speaks on cloud security at major industry events like Microsoft Future Decoded and the European Information Security Summit.
+
+David is married to a lovely girl who makes the best banana cake in the world. They love travelling the world together and intend to do missions in Asia very soon!
+*
```

Comment:
Is this intended?

Reply:
Think this is for italics on the speaker bio but we'll need them per paragraph

Reply:
> Is this intended?

is what intended, i don't see what you mean .. sorry :-)

Reply:
just changed it in the files added to the PR, this om now?

Reply:
There, fixed it!